test/bin/scenario.sh

```diff
@@ -772,6 +772,13 @@ run_tests() {
     fi
     record_junit "${vmname}" "robot_framework_installed" "OK"
 
+    # Make sure oc command is available
+    if ! command -v oc &> /dev/null
+    then
+        error "OpenShift Client package not installed, install it with ${ROOTDIR}/scripts/fetch_tools.sh oc"
+        exit 1
```

Comment:
Please include the junit entry (e.g. L770):
```
record_junit "${vmname}" "robot_framework_installed" "FAILED"
```

Reply:
Included, don't know why this was not marked as outdated